lambda/downsample_volume.py

```diff
@@ -75,10 +76,11 @@ def get(self, key):
         try:
             resp = self.s3.get_object(Key = key,
                                       Bucket = self.bucket)
-        except:
-            raise Exception("No Such Key")
-
-        self._check_error(resp, "reading")
+        except ClientError as e:
```

Comment:
Would it be nicer to use `except self.s3.exceptions.NoSuchKey as e:`?

Reply:
I tried that initially but it didn't seem to work (be caught). It could have been the effect of using `moto`.